src/db/sqlite/lrsql/sqlite/sql/ddl.sql

```diff
@@ -328,3 +328,41 @@ SET sql = 'CREATE TABLE credential_to_scope (
     ON DELETE CASCADE
 )'
 WHERE type = 'table' AND name = 'credential_to_scope'
+
+/* Migration 2022-05-08-00 - Add timestamp to xapi_statement */
+
+-- :name query-xapi-statement-timestamp-exists
+-- :command :query
+-- :result :one
+-- :doc Query to see if `xapi_statement.timestamp` exists.
+SELECT 1 FROM pragma_table_info('xapi_statement') WHERE name = 'timestamp'
+
+-- :name alter-xapi-statement-add-timestamp!
+-- :command :execute
+-- :doc Add `xapi_statement.timestamp` to allow easier timestamp access.
+ALTER TABLE xapi_statement ADD COLUMN timestamp TIMESTAMP
```

Comment:
I overlooked this but I didn't realize that SQLite had a `TIMESTAMP` type keyword. If you looked at the `last_modified` property elsewhere in the DDL you'll see I'll just have their type as `TEXT`.

Reply:
its a do-nothing alias, but in this case a useful one for inspection purposes

Reply:
What would happen if we changed the `TEXT` type in those instances with `TIMESTAMP` in the SQLite DDL?